src/edamame/impl/parser.cljc

```diff
@@ -328,7 +328,7 @@
               (parse-next ctx reader))
             (let [sym (parse-next ctx reader)
                   data (parse-next ctx reader)
-                  f (or (get (:readers ctx) sym)
+                  f (or ((:readers ctx) sym)
```

Comment:
I think this will cause NullPointer exception when :readers isn' set.

Reply:
Fixed in f16695d